.travis.yml

```diff
@@ -1,44 +1,12 @@
-notifications:
-  email: false
```

Comment:
Is this false by default, now?

Reply:
looks like by default it sends email (https://docs.travis-ci.com/user/notifications/#configuring-email-notifications) to the "committer and the author, but only if they have access to the repository the commit was pushed to"

my changeset reverts the file to how it was before I modified it to add docs testing. but I am happy to add this snippet back in if you think that is best (I am not familiar with what is typical here)

Reply:
I'd leave it in for now :slightly_smiling_face: